test/Be.Vlaanderen.Basisregisters.Aws.Lambda.Tests/Be.Vlaanderen.Basisregisters.Aws.Lambda.Tests.csproj

```diff
@@ -4,5 +4,10 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Be.Vlaanderen.Basisregisters.Aws.Lambda\Be.Vlaanderen.Basisregisters.Aws.Lambda.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <None Update="appsettings.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
```

Comment:
needed?

Reply:
yes, because the test will start the lambda and in the `ConfigureService` the appsettings.json is marked as required

Reply:
You can create your own settings in a test. This doesn't look like an integration test.
example:
``` var configuration = new ConfigurationBuilder()
                .AddInMemoryCollection(new Dictionary<string, string> { { "ConnectionStrings:Events", "x" } })
                .Build();```